### PR TITLE
[ACS] only expose orchestrator-release for version 2017-07-01 or above

### PR DESCRIPTION
**Pull request by rjtsdl**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [x] Each command and parameter has a meaningful description.
- [x] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

**Comment by rjtsdl** [comment]
this is a patch to previous #4421

**Comment by rjtsdl** [comment]
@derekbekoe to review and merge

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py [hunk 1]
```diff
@@ -740,6 +739,8 @@ def _create(resource_group_name, deployment_name, dns_name_prefix, name, ssh_key
             "adminUsername": admin_username
         },
     }
+    if api_version == "2017-07-01":
```
Comment (derekbekoe) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py]: What happens when there's a new API version released? Will this list have to be updated?
The method `supported_api_version` at `azure.cli.core.profiles` may be useful here?

Or is this really only for *this* specific API version

Comment (rjtsdl) [reply src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py]: currently, we are in a very tricky stage. we have two total different RP both running. However, we expect to see only one RP remaining after 2 months. 

In the long term, we shouldn't have to specify api-version. Currently, we want to expose the api-version, only supported by one type of RP to preview clients.

For short, it is temporary.

**Pull request merged**